packages/core/core/src/plugin.ts

```diff
@@ -138,10 +138,6 @@ export class Plugin {
     this._loggerObject[type](this.name + ':', ...[].slice.call(arguments, 1));
   }
 
-  setUpLogger() {
-    this.logger = new Logger({});
-  }
```

Comment:
This removed the events from the Logger and doesn't even intercept the logs correctly. i.e this doesn't do anything any more